apollo-adminservice/src/main/resources/application-zookeeper-discovery.properties

```
@@ -0,0 +1,23 @@
+#
+# Copyright 2021 Apollo Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apollo.eureka.server.enabled=false
+eureka.client.enabled=false
+
+#zookeeper enabled
+spring.cloud.zookeeper.enabled=true
+spring.cloud.zookeeper.discovery.enabled=true
+spring.cloud.zookeeper.discovery.register=true
+spring.cloud.zookeeper.discovery.instance-id=apollo-adminservice
```

I believe the instance id should differ from different instances, or they might override each other.
```suggestion
spring.cloud.zookeeper.discovery.instance-id=${spring.cloud.client.ip-address}:${server.port}
```

---

已修改